src/sentry/migrations/0481_add_pull_request_comment_table.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 2.2.28 on 2023-06-07 21:44
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import sentry.db.models.fields.array
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0480_remove_organization_user"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="PullRequestComment",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("external_id", sentry.db.models.fields.bounded.BoundedBigIntegerField()),
+                ("created_at", models.DateTimeField()),
+                ("updated_at", models.DateTimeField()),
+                ("issues", sentry.db.models.fields.array.ArrayField(null=True)),
+                (
+                    "pull_request",
```

Comment:
I assume we're going to fetch the comments per pr very frequently. Do you want to add an index for this field to make that faster?